cluster/config-defaults.yaml

```diff
@@ -757,6 +757,7 @@ spot_allocation_strategy: "capacity-optimized"
 stackset_controller_sync_interval: "10s"
 stackset_controller_mem_min: "120Mi"
 stackset_controller_mem_max: "4Gi"
+stackset_hpa_field_enabled: "true"
```

Comment:
It could be a good idea to set it `false` by default and then set it `true` on _all_ clusters. Then deleting the config item per cluster is the same as disabling the feature.

Maybe also call this config item `stackset_legacy_hpa_field_enabled` to be consistent with the naming in the admission-controller config.